docs/labs/sql-injection.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="https://best.openssf.org/assets/css/style.css">
+<link rel="stylesheet" href="checker.css">
+<script src="js-yaml.min.js"></script>
+<script src="checker.js"></script>
+<link rel="license" href="https://creativecommons.org/licenses/by/4.0/">
+
+<!-- See create_labs.md for how to create your own lab! -->
+
+<!-- Sample expected answer -->
+<script id="expected0" type="plain/text">String QueryString = "select * from authors where lastname = ?";</script>
+
+<script id="correct0" type="plain/text">
+ (String QueryString)\s\=\s\"select\s+\*\s+from\s+authors\s+where\s+lastname\s\=\s+\?\"\;
+</script>
+
+<!-- Full pattern of correct answer -->
+  <!-- 
+<script id="correct0" type="plain/text">
+  ^[?]$ 
+</script>
+-->
+
+<script id="expected1" type="plain/text">rs = statement.executeQuery(QueryString,lastname);</script>
+
+<script id="correct1" type="plain/text">
+  \s* rs = statement \. executeQuery \( QueryString, \s* lastname \) ; \s*
```

Comment:
Overall looks good. The `\s*` before lastname is unnecessary; a single space has the same meaning and is simpler.